features/data_converter/empty/feature.py

```diff
@@ -0,0 +1,66 @@
+import os
+from datetime import timedelta
+from typing import Optional
+
+from google.protobuf.json_format import Parse
+from temporalio import activity, workflow
+from temporalio.api.common.v1 import Payload
+from temporalio.api.enums.v1 import EventType
+from temporalio.client import WorkflowHandle
+from temporalio.exceptions import ApplicationError
+
+from harness.python.feature import Runner, register_feature
+
+
+@activity.defn
+async def null_activity(input: Optional[str]) -> None:
+    # check the null input is serialized correctly
+    if input != None:
+        raise ApplicationError("Activity input should be None")
```

Comment:
Is this non-retryable?

Reply:
nit: All of the application failures in this feature _should_ be non-retryable to prevent needless retries.

Reply:
no reason for it to be retryable, I'll update